src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,9 @@
+# Liquibase properties to create diff between local database and JPA entities
+url=jdbc:postgresql://localhost:5432/master-diet-db
```

Comment:
Can you parametrize it in master-diet-secrets in order to be able to use that in production?

Reply:
That goes for url, username and password

Reply:
It turns out that these files are useless, liquibase uses properties from application.yml

Reply:
Ok, cool